index.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module '@discoveryjs/json-ext' {
+    import { Readable } from 'stream';
+
+    export function parseChunked<T>(stream: Readable | Generator<any, void, unknown>): Promise<T>;
```

Comment:
no need any generic (`T`) here

Reply:
Function should return `Promise<string>`. It would be nice to add typings on generator w/o any/unknown if possible